packages/rsa-image-pcd-ui/src/CardBody.tsx

```diff
@@ -1,5 +1,10 @@
 import { styled } from "@pcd/passport-ui";
-import { RSAImagePCD } from "./RSAImagePCD";
+import { PCDUI } from "@pcd/pcd-types";
+import { RSAImagePCD } from "@pcd/rsa-image-pcd";
+
+export const RSAImagePCDUI: PCDUI = {
```

Comment:
I wonder if `PCDUI` in each of these packages should be more definitely defined by passing in the particular claim and proof types?